packages/invoices/src/OutboundShipment/api/operations.graphql

```diff
@@ -1,3 +1,55 @@
+fragment PartialLocation on LocationNode {
+  id
+  name
+  code
+  onHold
+}
+
+fragment PartialStockLine on StockLineNode {
+  id
+  itemId
+  availableNumberOfPacks
+  totalNumberOfPacks
+  onHold
+  sellPricePerPack
+  packSize
+  expiryDate
+  location {
+    ...PartialLocation
+  }
+}
+
+fragment PartialItem on ItemNode {
+  id
+  name
+  code
+  unitName
+}
+
+fragment PartialOutboundLine on InvoiceLineNode {
```

Comment:
🤔  We have `PartialOutboundLineFragment` and `OutboundShipmentLineFragment`
Could one replace the other? 
Should `OutboundShipmentLineFragment` be renamed to `OutboundLineFragment`?
Does `Partial...` indicate a subset or is it a new naming convention for fragments?

Reply:
My thinking was the `Partial` fragment was to make the `DraftOutboundLine` creation easier.

Buuut. I'm thinking that the `OutboundShipmentLine` has too much stuff and we can probably use the one type 😁 

i've almost finished off the inbound side of changes so I'm going to take a look at that after and do the renaming also